injective/assetlist.json

```diff
@@ -219,6 +219,52 @@
       "logo_URIs": {
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/WOSMO.png"
       }
+    },
+    {
+      "description": "GLTO-ERC20 on injective",
+      "denom_units": [
+        {
+          "denom": "ibc/072E5B3D6F278B3E6A9C51D7EAD1A737148609512C5EBE8CBCB5663264A0DDB7",
+          "exponent": 0
+        },
+        {
+          "denom": "glto",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/072E5B3D6F278B3E6A9C51D7EAD1A737148609512C5EBE8CBCB5663264A0DDB7",
+      "name": "Gelotto",
+      "display": "glto",
+      "symbol": "injective.GLTO",
```

Comment:
You're OK to keep the symbol for GLTO on Injective as "GLTO". (The "injective.- prefix would be enforced only on Osmosis)